memgpt/config.py

```diff
@@ -106,6 +107,7 @@ def load(cls) -> "MemGPTConfig":
 
             # read config values
             model = config.get("defaults", "model")
+            context_window = config.get("defaults", "context_window")
```

Comment:
backwards compat for older agent configs:
`config.get("defaults", "context_window") if config.has_option("defaults", "context_window") else <some default value>`